pkgs/top-level/all-packages.nix

```diff
@@ -19120,4 +19120,6 @@ with pkgs;
   biblioteca = callPackage ../by-name/bi/biblioteca/package.nix {
     webkitgtk = webkitgtk_6_0;
   };
+
+  wivrn = qt6Packages.callPackage ../applications/video/wivrn { };
```

Comment:
Instead of using qt6Packages like this, you can keep wivrn a by-name package and just pull in `qt6` as an input.

Reply:
Good to know, I will try to move it back